src/webview/git-import/app/gitImport.tsx

```diff
@@ -469,7 +469,7 @@ export class GitImport extends React.Component<DefaultProps, {
                                     {!isDevFile &&
                                         <div className='strategyContainer strategyWarning'>
                                             <ErrorIcon color='warning' style={{ fontSize: 20 }} />
-                                            <Typography variant='body2' style={{ margin: '0 5px' }}>Here is the recommended devfile:</Typography>
+                                            <Typography variant='body2' style={{ margin: '0 5px' }}>Here {compDescription?.length > 1 ? 'are' : 'is' } the recommended devfile:</Typography>
```

Comment:
If there is more than one devfile shown, then the user needs to select a devfile. I think you should add the text "Please select a devfile that corresponds to the language/framework that you are using", or something similar, to indicate this to the user.